subsys/net/l2/ieee802154/ieee802154_mgmt.c

```diff
@@ -258,26 +272,34 @@ enum net_verdict ieee802154_handle_mac_command(struct net_if *iface,
 			return NET_DROP;
 		}
 
+		k_sem_take(&ctx->ctx_lock, K_FOREVER);
+
+		enum net_verdict ret = NET_DROP;
```

Comment:
kind of ugly to see a variable declaration it a middle of a code block

Reply:
Fixed.